### DISCUSSION TITLE
How to raise chart resolution in quarto publication?

### DISCUSSION THREAD

**Discussion by PhilSmith26** [quarto-dev/quarto-cli#1957] [Q&A] [answered]
I am producing a rather detailed facet chart. In RStudio it renders really well, but when I generate and then publish it in quarto, using the same code, it has lower resolution and does not look good. How do I raise the quarto resolution?

**Comment by mcanouil** [comment, marked as answer]
By setting `fig-dpi` for example.
You can see all figures options to make your figures as you want in <https://quarto.org/docs/reference/formats/html.html#figures>
Or for pdf format in 
<https://quarto.org/docs/reference/formats/pdf.html#figures>.
Note that a lot (if not all) options are shared across formats.

**Reply by PhilSmith26** [reply]
Thanks so much. That does the trick.
